app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,52 @@
+// COLORS
```

Comment:
We had individual stylesheets for these different sets of variables. I decided to centralize them all into a `variables` sheet to follow the conventions I'm seeing.